.teamcity/Gradle_Check/configurations/FunctionalTest.kt

```diff
@@ -36,10 +36,15 @@ class FunctionalTest(model: CIBuildModel, uuid: String, name: String, descriptio
     params {
         param("env.JAVA_HOME", "%${testCoverage.os}.${testCoverage.buildJvmVersion}.openjdk.64bit%")
         when (testCoverage.os) {
-            Os.linux -> param("env.ANDROID_HOME", "/opt/android/sdk")
+            Os.linux -> {
+                param("env.ANDROID_HOME", "/opt/android/sdk")
+            }
             // Use fewer parallel forks on macOs, since the agents are not very powerful.
-            Os.macos -> param("maxParallelForks", "2")
-            else -> {
+            Os.macos -> {
+                param("maxParallelForks", "2")
```

Comment:
Why don't we set an Android home on macOS as well?

Reply:
Since I only want to deal with Windows now. I installed the sdk on mac as well.

Do you want me to check if the test run on mac as well?

Reply:
Not necessarily, just missed the thing for symmetry's sake. If it's not a lot of trouble it would probably be a good idea to do it. If it isn't trivial, just leave it I'd say.

Reply:
I'll try it out now.

Reply:
Looks like it works: https://builds.gradle.org/viewLog.html?buildId=28809040&buildTypeId=Gradle_Check_Platform_14_instantExecution&tab=buildResultsDiv&branch_Gradle_Check_Platform_14=wolfs%2Fandroid-on-windows

Reply:
🎉